### DISCUSSION TITLE
Customize quit command or prevent the end user from quitting using "cmd+q"

### DISCUSSION THREAD

**Discussion by rrhiralal** [IBM/mac-ibm-notifications#137] [General]
I've found that if an end user hits command+q, they are able to quit IBM Notifier. Additionally this returns 0, or main button pressed. Are there any plans to either cause a quit to return something else and or give the ability to set a custom quit command? I didn't see this mentioned anywhere else, I don't think I missed anything in the wiki.

**Comment by SMartorelli** [comment]
Sorry for late response on this but seems like Github hooks for discussions doesn't work properly haha. Anyway I know we already discussed this on Slack and there is the [BUG](https://github.com/IBM/mac-ibm-notifications/issues/138) open as well.
